### PR TITLE
[T] (quests) refined_storage

### PR DESCRIPTION
**Pull request by luo-2233**
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 1]
```diff
@@ -123,7 +123,7 @@
 		{
 			x: 2.0d
 			y: 6.0d
-			description: ["Basic processors allow you to craft the majority of things in the mod!"]
+			description: ["基础处理器允许你制作Mod中的大多数东西！"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["基础处理器允许你制作本模组中的大多数东西！"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 2]
```diff
@@ -296,7 +296,7 @@
 		{
 			x: -1.5d
 			y: 10.5d
-			description: ["A storage block allows for the storage of items depending on the size! This is significant because it is the equivalent to a large chest! The disadvantage of this is it takes up space within the world! Though it may be cheaper."]
+			description: ["一个存储方块允许你基于它的容量存储定量的物品！这很重要，因为它相当于一个大箱子！这样做的缺点是它占用了世界内的空间！虽然它可能更便宜。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/refined_storage.snbt:299]:
```suggestion
			description: ["一个存储方块允许你基于它的容量存储一定的物品！这很重要，因为它相当于一个大箱子！这样做的缺点是它占用了世界内的空间！虽然它可能更便宜。"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 3]
```diff
@@ -33,7 +33,7 @@
 		{
 			x: 2.0d
 			y: 0.0d
-			description: ["This allows you to charge many items! Including making charged certus quartz!"]
+			description: ["这个机器允许你给一些物品充能！同时包括充能赛特斯石英！"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["这个机器允许你给一些物品充能！包括充能赛特斯石英水晶！"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt (hunk 1 above)
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["基础处理器允许你制作模组中的大多数东西！"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 4]
```diff
@@ -610,7 +610,7 @@
 		{
 			x: 7.0d
 			y: 9.5d
-			description: ["This allows you to encode patterns to be processed! Make sure to check the processing box to allow you to interface and use machines."]
+			description: ["他允许你对要处理的样板进行编码！确保选中处理框以允许你使用接口和机器。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["它允许你对要处理的样板进行编码！确保选中处理框以允许你使用接口和机器。"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 5]
```diff
@@ -737,7 +737,7 @@
 		{
 			x: 10.0d
 			y: 9.5d
-			description: ["The security manager allows you to manage your users and tell the system what exactly they are allowed to do!"]
+			description: ["这个权限管理器允许你管理存储系统的使用者并且告诉存储系统他们能做什么！"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/refined_storage.snbt:740]:
```suggestion
			description: ["这个权限管理器允许你管理存储系统的使用者并且告诉存储系统它们能做什么！"]
```

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 6]
```diff
@@ -495,7 +495,7 @@
 		{
 			x: 1.0d
 			y: 11.0d
-			description: ["This allows for storage of other inventories! Like a system of drawers or a colossal chest!"]
+			description: ["这允许你访问其他容器！就像抽屉系统或巨大的箱子一样！"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["这允许你访问其它容器！就像抽屉系统或巨大的箱子一样！"]
```

**Review by AnzhiZhang — APPROVED** [review]

File: config/ftbquests/quests/chapters/refined_storage.snbt [hunk 7]
```diff
@@ -737,7 +737,7 @@
 		{
 			x: 10.0d
 			y: 9.5d
-			description: ["The security manager allows you to manage your users and tell the system what exactly they are allowed to do!"]
+			description: ["这个权限管理器允许你管理存储系统的使用者并且告诉存储系统它们能做什么！"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/refined_storage.snbt]:
```suggestion
			description: ["这个权限管理器允许你管理存储系统的使用者并且告诉存储系统他们能做什么！"]
```
指玩家

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**